siaskynet/skynet.py

```diff
@@ -13,7 +14,11 @@ def uri_skynet_prefix():
     @staticmethod
     def default_upload_options():
         return type('obj', (object,), {
-            'portal_url': 'https://siasky.net',
+            'portal_url': [
```

Comment:
Change is technically sound, personally I feel this can be accomplished by wrapping the SDK method.
That way we don't have to maintain and curate a list of known portals in the SDK.
I'll make sure to get some more opinions on this.

Thanks for your contribution!

Reply:
Thanks so much peterjanbrone.  I'll close this PR for now.  When you have time I found the metadata leaves out length for some uploads and made #16 .